vendor/github.com/aws/aws-sdk-go/service/elasticache/api.go

```diff
@@ -2572,6 +2572,9 @@ type CacheCluster struct {
 	// node.
 	ConfigurationEndpoint *Endpoint `type:"structure"`
 
+	// The DNS name of the cache cluster without the port appended.
+	ClusterAddress *string `type:"string"`
```

Comment:
Ah! That change will be over written the next time that we updat ethe AWS SDK - I will manually merge your PR here and make the necessary changes :)